### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Isabellanamutenda**
- I added the contact form according to the instructions
- I styled the mobile version
- Add a REAdME file

### CODE REVIEW

**Comment by mimipeshy** [comment]
## Hi @Isabellanamutenda and @mwanawabangona 👋🏾 👋🏾 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a README file making your PR unprofessional. Add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by mikenath223** [comment]
Hi , 

You made a great effort in this project, but unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that will be invalidated for this reason:
-  Kindly ensure all GitHub linting checks pass.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding. Cheers!🚀🚀

If something isn't 100% clear, you can leave comments or questions in this PR thread.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **don't count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -62,11 +62,58 @@ <h5>Skills</h5>
...
+          <path d="M84.2067 23.4103C80.7385 17.3223 75.8692 12.1276 69.9932 8.24664C64.1172 4.36571 57.3996 1.9077 50.3842 1.07163C43.3689 0.235572 36.2531 1.04496 29.6129 3.43425C22.9728 5.82355 16.9949 9.72559 12.1634 14.8244C7.33183 19.9233 3.78242 26.0756 1.80255 32.7833C-0.177326 39.491 -0.532025 46.5655 0.767174 53.4339C2.06637 60.3023 4.98295 66.7717 9.28075 72.3182C13.5786 77.8646 19.1368 82.3323 25.5054 85.3594L33.8939 68.1011C30.2658 66.3766 27.0992 63.8314 24.6508 60.6716C22.2023 57.5117 20.5408 53.8261 19.8006 49.9132C19.0605 46.0003 19.2625 41.9699 20.3905 38.1486C21.5184 34.3272 23.5405 30.8222 26.293 27.9174C29.0455 25.0126 32.4511 22.7896 36.234 21.4285C40.0169 20.0673 44.0708 19.6062 48.0674 20.0825C52.064 20.5588 55.891 21.9591 59.2386 24.1701C62.5862 26.381 65.3602 29.3405 67.336 32.8088L84.2067 23.4103Z" fill="white"/>
+          </svg>
+    </div>
+    <h6>contact me</h6>
+    <p class="content">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <div class="form">
+      <form action="#" method="post">
```
Comment (arslanbisharat) [inline index.html:86]: - [ ] Please make sure the form uses the `action` attribute to integrate with formspree service & `action` value should be similar to `https://formspree.io/f/{form_id}`

File: index.html [hunk 2]
```diff
@@ -62,11 +62,58 @@ <h5>Skills</h5>
...
+                  <circle cx="16.2283" cy="16.2188" r="15.5552" transform="rotate(48.9435 16.2283 16.2188)" fill="white"/>
+                  </svg>
+                  </div>
+                  <div class="rectangle7">
+                      <svg width="70" height="58" viewBox="0 0 70 58" fill="none" xmlns="http://www.w3.org/2000/svg">
+                          <rect width="76.175" height="13.6273" transform="matrix(0.801365 0.598176 0.598176 -0.801365 0.804688 11.8887)" fill="#7F8CFF"/>
+                          </svg>
+                          
+                         </div>
+          <button type="submit">Get in touch</button>
```
Comment (arslanbisharat) [inline index.html:105]: - [ ] Make sure the project has button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi @Isabellanamutenda, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (Isabellanamutenda) [reply index.html:86]: Resolved

File: index.html (hunk 2 above)
Comment (Isabellanamutenda) [reply index.html:105]: Resolved

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**